css/css-ui-3/text-overflow-015.html

```diff
@@ -0,0 +1,29 @@
+﻿<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: ellipsis after graphical transformations</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#ellipsing-details">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem">
+<meta name="assert" content="Ellipsing occurs after relative positioning and other graphical transformations">
+<style>
+div {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100px;
+  height: 100px;
+  white-space: pre;
+  line-height: 1;
+  font-size: 20px;
+  font-family: ahem;
+  color: green;
+  background: linear-gradient(red, red) top right / 1em 1em no-repeat, green;
```

Comment:
Please replace this with swatch-red.png, as we don't want to be depending on css3-images if we don't need to.